code/035/script.js

```diff
@@ -0,0 +1,16 @@
+function User(username) {
+    this.username = username;
+  
+    this.updateUsername = (newName) => {
+      this.username = newName;
+    };
+  }
+  
+  const user1 = new User('kirandash');
+  const user2 = new User('bgwebagency');
+  
+  user1.updateUsername('openai');
+  user2.updateUsername('gpt-3');
+  
+  console.log(user1.username + user2.username);
```

Comment:
Please update this code to match with code from README.md file

```diff
- function User(username) {
-     this.username = username;
-   
-     this.updateUsername = (newName) => {
-       this.username = newName;
-     };
-   }
-   
-   const user1 = new User('kirandash');
-   const user2 = new User('bgwebagency');
-   
-   user1.updateUsername('openai');
-   user2.updateUsername('gpt-3');
-   
-   console.log(user1.username + user2.username);
  
+ function User(username) {
+   this.username = username
+   this.updateUsername = (newName) => {
+     this.username = newName
+   }
+ }
+
+ const user1 = new User('kirandash')
+ const user2 = new User('bgwebagency')
+
+ user1.updateUsername('-website')
+ user2.updateUsername('-app')
+
+ console.log(user1.username + user2.username)
```

Reply:
Thank You!
I will make the changes.